content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -87,6 +87,8 @@ For example:
         ...
 
 A given Kubernetes server will only preserve a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default.  When the requested watch operations fail because the historical version of that resource is not available, clients must handle the case by recognizing the status code `410 Gone`, clearing their local cache, performing a list operation, and starting the watch from the `resourceVersion` returned by that new list operation. Most client libraries offer some form of standard tool for this logic. (In Go this is called a `Reflector` and is located in the `k8s.io/client-go/cache` package.)
+
+{{< feature-state for_k8s_version="v1.16" state="beta" >}} 
```

Comment:
another nit, totally optional:
You could add a sub-header, such as:
### Watch bookmarks

And then below the sub-header add the feature-state shortcode, followed by a new line and the rest of the content (see if this looks ok).

Reply:
done